sdk/nodejs/service/index.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as runtime from "../runtime";
+
+export function getStack(name: string): Promise<any> {
```

Comment:
We should make this more strongly typed.  But I'm actually not sure - is the result a `Record<string, string>`, or can the values be richer values?

Reply:
The values can be richer, yes (e.g. many examples export `kubeconfig` as a POJO rather than a string).

Reply:
I know you can export a POJO - but I wasn't sure whether there is any step in the process here where we serialize this down into a string and then don't de-serialize back.  I assume not, and that these are ultimately just read from the PropertyMap in the checkpoint on the stack resource.

Reply:
> these are ultimately just read from the PropertyMap in the checkpoint on the stack resource.

Correct.